messaging/kafkaSSL/src/main/java/io/quarkus/qe/messaging/ssl/quickstart/KafkaEndpoint.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.qe.messaging.ssl.quickstart;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+public abstract class KafkaEndpoint {
+
+    public static final String TOPIC = "hello";
+    volatile boolean done = false;
+    volatile String last;
+
+    protected void initialize(KafkaConsumer<String, String> consumer) {
+        consumer.subscribe(Collections.singleton(TOPIC));
+        new Thread(() -> {
+            while (!done) {
+                final ConsumerRecords<String, String> consumerRecords = consumer.poll(Duration.ofSeconds(1));
+
+                consumerRecords.forEach(record -> {
+                    System.out.printf("Polled Record:(%s, %s, %d, %d)\n",
```

Comment:
Could you please summarize pros to using SOP over logger in this case?

Reply:
Thanks were a leftover statement. Quarkus logs allow you to filter logs based on your environment configuration, so is always more convience. System out is the fast&furious way :)

Doc Ref:  https://quarkus.io/guides/logging#runtime-configuration